NEX-33 // Update footer

mark-a-wilson opened the pull request:
- Remove street address
- Remove links we're not sure about including yet or not
- simplify styling

mark-a-wilson committed:
Update styles for footer

mark-a-wilson committed:
Update footer markup

- Remove street address
- Revert to default Bootstrap structure and classes

parulmishra commented:
SonarQube Analysis: https://sonarqube-ifttgq-tools.pathfinder.gov.bc.ca/dashboard?id=BCDevExchange-PR-43

thegentlemanphysicist reviewed (dismissed):
Looks good.

mark-a-wilson committed:
Merge branch 'develop' into NEX-33

mark-a-wilson committed:
change footer nav-link color and size

mark-a-wilson committed:
more tweaks

- remove old footer logo
- incorporate "Supported" logo
- make logo and nav links centered on sm, justified on md

mark-a-wilson committed:
restyle footer

mark-a-wilson committed:
Merge branch 'develop' into NEX-33

thegentlemanphysicist reviewed (commented):
Looks good, tested responsive styling on the browser of the nav bar

mark-a-wilson committed:
Merge branch 'develop' into NEX-33
